content/docs/2.0/scalers/mysql.md

```diff
@@ -34,6 +34,7 @@ Some parameters could be provided using environmental variables, instead of sett
 
 - `hostFromEnv` - The host of the MySQL server, similar to `host`, but reads it from an environment variable on the scale target.
 - `portFromEnv` - The port of the MySQL server, similar to `port`, but reads it from an environment variable on the scale target.
+- `usernameFromEnv` - The username for connecting with host of the MySQL server, similar to `username`, but reads it from an environment variable on the scale target.
```

Comment:
Move this change to the version 2.15 please. This feature will be included as part of it

Reply:
Thanks @JorTurFer . Have addressed it